geopandas/tests/test_geoseries.py

```diff
@@ -236,6 +240,22 @@ def test_proj4strings(self):
         reprojected_dict = self.g3.to_crs({"proj": "utm", "zone": "30N"})
         assert np.all(reprojected_string.geom_almost_equals(reprojected_dict))
 
+    def test_covers_itself(self):
```

Comment:
Small additional request: could you move those tests to `test_geom_methods.py`? (it's there that the other predicate tests are as well for within etc)

You don't need to rewrite them (as you will see the tests like `test_within` follow a bit a different style, testing the result as an array), as yours are actually better. But then we keep all predicate tests in the same place (and can clean-up the others later)